Modules/Networking/Requests/DummyIngestionRequest.swift

```diff
@@ -0,0 +1,47 @@
+// DummyIngestionRequest.swift
+// Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+// Please refer to the AUTHORS file for more information.
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import Alamofire
+import Extensions
+import Foundation
+import Models
+
+public struct DummyIngestionRequest: FixedSizeJSONRequest {
```

Comment:
out of curiosity: why a separated request rather than using the same as the "real" one?

Reply:
Semantics. I see this more as a third type of request that has the same "weight" of the other two. I don't have a strong opinion, though.

Reply:
GTM :)